src/delegation_backend/app_config.go

```diff
@@ -36,9 +36,22 @@ func LoadEnv(log logging.EventLogger) AppConfig {
 		}
 	} else {
 		networkName := getEnvChecked("CONFIG_NETWORK_NAME", log)
-		gsheetId := getEnvChecked("CONFIG_GSHEET_ID", log)
-		delegationWhitelistList := getEnvChecked("DELEGATION_WHITELIST_LIST", log)
-		delegationWhitelistColumn := getEnvChecked("DELEGATION_WHITELIST_COLUMN", log)
+
+		delegationWhitelistDisabled := os.Getenv("DELEGATION_WHITELIST_DISABLED")
+		var gsheetId, delegationWhitelistList, delegationWhitelistColumn string
+		if delegationWhitelistDisabled == "true" {
```

Comment:
This condition is tricky, because setting the variable to `True` or `TRUE` or `1` won't work as expected. For this reason in the uptime analyzer I specifically require that this variable is set to `0` or `1` or unset. Any other value causes an error. I suggest to do the same here. Actually, perhaps you can reuse the function I wrote to do this here.

Reply:
Ok, fixed. Couldn't re-use your func as go complained about circular dependency issue between packages. Extracting the func to separate package turns not an option because the AppConfig types are different between packages atm. So came up with my own little func to check this.